### DISCUSSION TITLE
How to Use Overture in GraphHopper?

### DISCUSSION THREAD

**Discussion by larsskaug** [OvertureMaps/data#289] [Discussion]
As a developer of navigation software that accounts for road risk, I need to load Overture data into GraphHopper. 

In this example of [how to match GPS graces to segments](https://github.com/OvertureMaps/schema/blob/dev/gers/examples/python/MATCH_TRACES.md), `ibnt1` and `jenningsanderson` mention "converting the Overture data set to OSM format, loading it in one of the routing engines available and use the map matching services available."

Can anyone provide an example of how this can be done? The only solution I can think of is to create an `OPL` file with custom code which I would then turn into a `OSM.PBF` file using `osmium` but that is error-prone,  and cumbersome.

Thanks!

**Comment by brad-richardson** [comment]
@larsskaug we don't have any docs around this, but routing is a current area of focus. My opinion is that built-in support with a router like GraphHopper, Valhalla, etc would be the best option and we don't plan to support any OSM conversion tools directly. Networkx support (similar to osmnx) is something I'm currently looking into, would that help fill any gaps for you?

If you're interested in building one of these integrations, let me know so we can see if we can help!
